### ISSUE TITLE
Feature request: allow totally public editing

### ISSUE THREAD

**Issue by amandafrench** [boonebgorges/participad#35]
I love that Participad is tied to WP users, especially on our multisite / buddypress network, but it'd be useful to be able to declare that anybody and their tabby cat can edit a particular Notepad without signing in or anything. 

I had thought that creating a Notepad on the thatcamp.org blog, which is the primary blog for our multisite / BP network, might enable anyone with a thatcamp.org account to edit that Notepad, but I tested it, and no. That would also be excellent, but failing that, totally public would work.

**Comment by boonebgorges** [comment]
Yup, this is a good idea. It'll mean rewriting some of the permissions
structure - as it stands, access is strictly controlled based on a
cookie linked to user ID, and I'd have to find a way to fake it for
non-logged-in users - but I think it's a worthwhile feature.
